scripts/pre-release.sh

```diff
@@ -16,6 +17,15 @@ function cleanup {
     delete_file spartacussampledata.2105.zip
     delete_file spartacussampledata.2105.tar.gz
 
+    delete_file epdvisualizationspartacussampledata.1905.zip
+    delete_file epdvisualizationspartacussampledata.1905.tar.gz
+    delete_file epdvisualizationspartacussampledata.2005.zip
+    delete_file epdvisualizationspartacussampledata.2005.tar.gz
+    delete_file epdvisualizationspartacussampledata.2011.zip
+    delete_file epdvisualizationspartacussampledata.2011.tar.gz
```

Comment:
Hi Sam. I believe none of these files are created in the first place?

Reply:
That's right. The intention would be to add them once someone has tested the extension with those commerce cloud versions.
I will remove these lines.